deployments/gpu-operator/templates/cleanup_crd.yaml

```diff
@@ -35,16 +35,12 @@ spec:
           image: {{ include "gpu-operator.fullimage" . }}
           imagePullPolicy: {{ .Values.operator.imagePullPolicy }}
           command:
-          - sh
-          - -c
-          - >
-              kubectl delete clusterpolicy cluster-policy;
-              kubectl delete crd clusterpolicies.nvidia.com;
-              kubectl delete crd nvidiadrivers.nvidia.com --ignore-not-found=true;
-            {{- if .Values.nfd.enabled -}}
-              kubectl delete crd nodefeatures.nfd.k8s-sigs.io --ignore-not-found=true;
-              kubectl delete crd nodefeaturegroups.nfd.k8s-sigs.io --ignore-not-found=true;
-              kubectl delete crd nodefeaturerules.nfd.k8s-sigs.io --ignore-not-found=true;
-            {{- end }}
+            - /usr/bin/apply-crds
```

Comment:
Shouldn't we be invoking the `delete` subcommand?

Reply:
Yes! :) I accidentally deleted it when I made formatting changes.